dependencies/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM --platform=${BUILDPLATFORM:-linux/amd64} debian:bookworm-slim AS build
+ARG TARGETPLATFORM
+
+# See for details: https://github.com/hadolint/hadolint/wiki/DL4006
+SHELL ["/bin/bash", "-euo", "pipefail", "-c"]
+
+RUN mkdir -p /tmp/build
+WORKDIR /tmp/build
+
+COPY prepare.sh /tmp/build/
+RUN ./prepare.sh
+
+COPY build_libraw.sh /tmp/build/
+RUN export $(cat /env) && ./build_libraw.sh
+COPY build_libheif.sh /tmp/build/
+RUN export $(cat /env) && ./build_libheif.sh
+COPY build_imagemagick.sh /tmp/build/
+RUN export $(cat /env) && ./build_imagemagick.sh
+COPY download_jellyfin-ffmpeg.sh /tmp/build/
+RUN export $(cat /env) && ./download_jellyfin-ffmpeg.sh
+
+COPY output.sh /
+RUN /output.sh
+
+FROM debian:bookworm-slim AS release
```

Comment:
While working on the follow-up PR, I found `scratch` annoying if I wanted to check the output of dependencies since it can't launch directly as there is no shell. So I changed to `debian:book-slim`. As this base image < 100MiB, I prefer to keep it for the inspection.